src/main/java/com/thebois/views/infoviews/InventoryView.java

```diff
@@ -0,0 +1,59 @@
+package com.thebois.views.infoviews;
+
+import java.util.Locale;
+import java.util.Map;
+
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.VerticalGroup;
+
+import com.thebois.models.inventory.items.ItemType;
+import com.thebois.utils.StringUtils;
+
+/**
+ * Information view for an inventory.
+ */
+public class InventoryView implements IActorView {
+
+    private static final float PADDING = 20f;
+    private final Skin skin;
+    private final VerticalGroup root;
+
+    /**
+     * Instantiates the view that renders inventory info.
+     *
+     * @param skin The skin to style widgets with.
+     */
+    public InventoryView(final Skin skin) {
+        this.skin = skin;
+
+        root = new VerticalGroup().pad(PADDING);
+        root.expand().fill();
+    }
+
+    @Override
+    public Actor getWidgetContainer() {
+        return root;
+    }
+
+    /**
+     * When new count is received the view is will update the counters for all items.
+     *
+     * @param inventoryItemCount The count of each item in the inventory.
+     */
+    public void update(final Map<ItemType, Integer> inventoryItemCount) {
+        root.clearChildren();
+        final Label defaultInventoryLabel = new Label("Inventory", skin);
+        root.addActor(defaultInventoryLabel);
+        for (final var entry : inventoryItemCount.entrySet()) {
+            final String labelName = entry.getKey().name().toLowerCase(Locale.ROOT);
+            final String labelText = StringUtils.capitalizeFirst(labelName + "s")
+                                     + ": "
+                                     + entry.getValue();
+            final Label inventoryInformationLabel = new Label(labelText, skin);
+            root.addActor(inventoryInformationLabel);
+        }
+    }
```

Comment:
style

I like descriptive names, both for variables and methods, but I'm not a fan of redundant names. This method lies in `InventoryView`, which means it's a view of an `Inventory`, and everything it deals with is related to an `Inventory` (unless you're breaking SRP, which you're not). 

Therefore it's, imo, redundant to have the word "inventory" in any variable or method name, since it's obviously related to inventory already. You wouldn't call `inventory.updateInventory()`, right?

E.g. 
* `inventoryItemCount` - what else would the item count be from? 
* `defaultInventoryLabel` - it's a `titleLabel`, no? `default` implies that it can take on other values, which in this case it can't. 
* `labelName` and `labelText` seem to describe how the variables are used, not *what* they contain. My preference is to name them after their actual values, so `itemName` and `itemNameAndCount`. I already know that it's text from the fact that they're `String`s, and I can already see they're being used in a label just below.
* `entry` - now this one is a bit more tricky, as it's a key-value-pair in an object. Naming this according to its value is hard as it'd have to be something like `itemTypeCount` which doesn't perfectly describe it. `entry` actually does point out that the variable is just an entry in a map. I'd leave this one be I think.

Reply:
👍